h5p-bildetema/src/components/Select/Select.module.scss

```diff
@@ -37,7 +37,7 @@
   padding: 0.25rem;
   position: absolute;
   margin-top: 3.5rem;
-  z-index: $fav-language-selector-z-index;
+  z-index: $select-z-index;
```

Comment:
Use this `z-index `within the `focus-visible` (line 52) also, to ensure that it will not be overwritten by other styles